deployments/sandbox/environment.py

```diff
@@ -21,27 +21,23 @@ def env() -> Mapping[str, Optional[str]]:
         # Set variables for the `sandbox` deployment here. The sandbox can be used to
         # run integration tests against a PR and to perform CI/CD experiments.
         #
-        # To define an AZUL_… variable use `_set AZUL_FOO bar`. For all other
-        # variables use `export BAR=baz`.
-        #
         # Only modify this file if you intend to commit those changes. To change the
         # environment with a setting that's specific to you AND the deployment, create a
-        # environment.local right next to this file and make your changes there. In that
-        # file the same rules apply with regard to `_set` vs `export`. Settings
-        # applicable to all environments but specific to you go into environment.local
-        # at the project root.
+        # environment.local right next to this file and make your changes there.
+        # Settings applicable to all environments but specific to you go into
+        # environment.local at the project root.
```

Comment:
```suggestion
        # environment.local.py at the project root.
```

Here and in the other copies. It's my bug but since you touched this …